appcast.xml

```diff
@@ -5,11 +5,24 @@
     <link>https://transmissionbt.com/appcast.xml</link>
     <description>Most recent version available.</description>
     <language>en</language>
-    <pubDate>Fri, 15 May 2020 20:00:00 -0400</pubDate>
+    <pubDate>Tue, 07 Feb 2023 20:00:00 -0400</pubDate>
     
     <item>
-      	<title>Transmission 3.00</title>
+      	<title>Transmission 4.00</title>
       	<sparkle:releaseNotesLink>https://transmissionbt.com/appcast/releasenotes.html</sparkle:releaseNotesLink>
+      	<pubDate>Tue, 07 Feb 2023 20:00:00 -0400</pubDate>
+      	<enclosure
+      	url="https://github.com/transmission/transmission-releases/raw/master/Transmission-4.00.dmg"
+      	sparkle:version="14714.4.0"
+      	sparkle:shortVersionString="4.00"
```

Comment:
```suggestion
      	sparkle:shortVersionString="4.0.0"
```

Although, not sure what it affects...

Also, should the above `14714.4.0` be `14714.4.0.0` as well? Probably doesn't matter for 4.0.0, but if/when we do 4.0.1 this will need to work somehow...

Reply:
I take that back, it should be `14714.4.0.0` to match the info.plist